src/application/DDD/EntryFactory.cpp

```diff
@@ -0,0 +1,25 @@
+#include "EntryFactory.h"
+#include "PlaintextPassword.h"
+#include "../InstanceManager.h"
+
+using namespace DDD::Factories;
+using namespace DDD::Entities;
+
+Entry EntryFactory::createEntry(unsigned int id, std::string entryNameString, std::string loginString, std::string encryptedPasswordString)
+{
+    DDD::ValueObjects::EntryId entryId(id);
```

Comment:
Seems odd to be using namespaces for Factories and Entities but not for ValueObjects. Also `using namespace DDD` I would use either way because `DDD::ValueObjects` is quite a bit.

Reply:
I did use all three relevant namespaces before, namely Entities, Factories and ValueObjects. However with all the ValueObjects the compiler said that the reference is ambiguous. Prefixing them with the namespace fixed it, so that's why ValueObjects was the only namespace that wasn't used. It did look weird though, I agree.
I'm now only using the DDD namespace and prefixed all classes with the more specific one.